matchzoo/preprocessors/bert_preprocessor.py

```diff
@@ -0,0 +1,141 @@
+"""Bert Preprocessor."""
+
+from tqdm import tqdm
+
+from . import units
+from .chain_transform import chain_transform
+from matchzoo import DataPack
+from matchzoo.engine.base_preprocessor import BasePreprocessor
+from .build_vocab_unit import built_bert_vocab_unit
+from .build_unit_from_data_pack import build_unit_from_data_pack
+
+tqdm.pandas()
+
+
+class BertPreprocessor(BasePreprocessor):
+    """Bert-base Model preprocessor."""
+
+    def __init__(self, fixed_length_left: int = 30,
+                 fixed_length_right: int = 30,
+                 filter_mode: str = 'df',
+                 filter_low_freq: float = 2,
+                 filter_high_freq: float = float('inf'),
+                 remove_stop_words: bool = False,
+                 lower_case: bool = True,
+                 chinese_version: bool = True,
+                 bert_vocab_path: str = "bert_resources/vocab.txt"):
+        """
+        Bert-base Model preprocessor.
+
+        TODO: doc here.
+
+
+        Example:
+            >>> import matchzoo as mz
+            >>> train_data = mz.datasets.toy.load_data()
+            >>> test_data = mz.datasets.toy.load_data(stage='test')
+            >>> # the argument 'bert_vocab_path' must feed the bert vocab path

```

Comment:
1. The first letter in the comment need to be upper case.
2. The argument `bert_vocab_path` must feed the bert vocab path, you mentioned **must**, then please check the argument in the function and remove default value.
3. It's better for us to unify the function-level indent, i,e,

```python
def __init__(
    self,
    fixed_length_left: int = 30,
    ...
)
```

4. Remove TODO.